www/schema-table.json

```diff
@@ -7,7 +7,8 @@
 				"PA": "Part A",
 				"PB": "Part B"
 			},
-			"Comments": "The whole is equal to the sum of two parts."
+			"Comments": "The whole is equal to the sum of two parts.",
+			"slot names": "Whole, Part A, Part B"
```

Comment:
As previously discussed, the slot names field is redundant with the Mapping.  Go ahead and remove all "slot names" fields.